chia/full_node/eligible_coin_spends.py

```diff
@@ -180,30 +167,7 @@ def get_deduplication_info(
                 # solution we see from the relatively highest FPC item, to avoid
                 # severe performance and/or time-complexity impact
                 raise SkipDedup("Solution is different from what we're deduplicating on")
-            # Let's calculate the saved cost if we never did that before
-            if duplicate_cost is None:
-                # See first if this mempool item had this cost computed before
-                # This can happen if this item didn't get included in the previous block
-                spend_cost = spend_data.cost
-                if spend_cost is None:
-                    spend_cost = run_for_cost(
-                        puzzle_reveal=spend_data.coin_spend.puzzle_reveal,
-                        solution=spend_data.coin_spend.solution,
-                        additions_count=len(spend_data.additions),
-                        max_cost=max_cost,
-                    )
-                    # Update this mempool item's coin spends map
-                    bundle_coin_spends[coin_id] = BundleCoinSpend(
-                        coin_spend=spend_data.coin_spend,
-                        eligible_for_dedup=spend_data.eligible_for_dedup,
-                        eligible_for_fast_forward=spend_data.eligible_for_fast_forward,
-                        additions=spend_data.additions,
-                        cost=spend_cost,
-                    )
-                duplicate_cost = spend_cost
-                # If we end up including this item, update this entry's cost
-                new_dedup_spends[coin_id] = DedupCoinSpend(current_solution, duplicate_cost)
-            cost_saving += duplicate_cost
+            cost_saving += dedup_coin_spend.cost
```

Comment:
`max_cost` is no longer needed (addressed already in addendum).

Reply:
I selected the diff block that made `max_cost` no longer needed. This interface didn't let me pick the line where this param is accepted.